handler/src/main/java/io/netty/handler/ssl/SslProvider.java

```diff
@@ -27,5 +29,9 @@
     /**
      * OpenSSL-based implementation.
      */
-    OPENSSL
+    OPENSSL,
+    /**
+     * OpenSSL-based implementation which does not have finalizers and instead implements {@link ReferenceCounted}.
+     */
+    OPENSSL_RELEASABLE
```

Comment:
@Scottmitch even better would be if we could also mark the `ReferenceCountedOpenSsl*` classes as unstable but I guess the annotation would then also be part of the sub-classes :(